proxy/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 #![cfg_attr(feature = "cargo-clippy", allow(clone_on_ref_ptr))]
 #![cfg_attr(feature = "cargo-clippy", allow(new_without_default_derive))]
-#![deny(warnings)]
+// #![deny(warnings)]
```

Comment:
Could this be reverted? This was only intended to be temporary.

Reply:
Yeah, totally forgot, thanks. AFAIK all the warnings on this branch have been dealt with.